main.nf

```diff
@@ -0,0 +1,32 @@
+// Ensure DSL2
+nextflow.enable.dsl = 2
+
+//test params
+params.config = "/agora-data-tools/test_config.yaml"
+
+//runs test config for Agora
+process AGORA_DATA_RUN {
+
+    debug true
+
+    container "sagebionetworks/agora-data-tools"
+
+    secret "SYNAPSE_AUTH_TOKEN"
+
+    input:
+    val(config)
```

Comment:
Shouldn't this be a `path` input value? 

```suggestion
    path config
```

I'm guessing you tried that first, but it wasn't working. That's because Tower looks for this file outside of the container so it can stage it within the container. I'm guessing this file is only available within the container. Try using the raw GitHub URL instead, which Nextflow will automatically download. This also ensures that you will always using the latest version of the test config (which might be a good thing or a bad thing). 

```nextflow
profiles {

    docker {
        ...
    }

    test {
        params {
            config = "https://raw.githubusercontent.com/Sage-Bionetworks/agora-data-tools/dev/test_config.yaml"
        }
    }

    debug {
        ...
    }

}
```

Reply:
You are exactly right. I will try with the GitHub URL